### ISSUE TITLE
Care to explain new features?

### ISSUE THREAD

**Issue by nebulous42069** [henryjfry/repository.thenewdiamond#18]
Hey bud, care to explain the Real Debrid Player options in settings? Curious to know. Thanks!

**Comment by nebulous42069** [comment]
Also Setup TMDBH Players?

**Comment by henryjfry** [comment]
Setup TMDBH players will setup the additional player files I created, so one is for RD playback (like realizer) and the other is for bluray playback, either directly playing the movie or with the interim kodi title select menu.

The Diamond RD player will scrape your cloud and play episodes/movies from your cloud.  So say you are watching a season pack, all those episodes are already cached so you can watch them straight from the RD cloud.
So basically I rebuilt a realizer type functionality.

However if it cant find the next episode it will scrape for it with seren. Then it will rescrape your cloud and queue it up.
Basically its only useful for episodes, i built it to do some pretty fuzzy episode matching as far as i am aware im the only person that uses it. So it may work for you it may not.

**Comment by nebulous42069** [comment]
Ok, well explained. Thank You.
